index.php

```diff
@@ -142,6 +142,15 @@
   <span>Load more</span>
 </div>
 
+<!-- <div class="modal modal--welcome">
```

Comment:
Since this is commented out but not deleted, does that imply it may be used at some point in the future? If it's gone permanently let's just remove it. If it's being kept for later, use a PHP comment rather than an HTML comment so it's not even served to the client (that will also solve the problems of the double-hyphens in the classes).